addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -217,25 +217,40 @@
 </template>
 
 <template id="s_mail_block_event" name="Event">
-    <div class="o_mail_snippet_general o_mail_block_event s_event bg-300">
+    <div class="o_mail_snippet_general s_mail_block_event bg-100 pt32 pb32" data-snippet="s_mail_block_event">
```

Comment:
You replaced the class `o_mail_block_event` with `s_mail_block_event`. I think that's a mistake.